makefile

```diff
@@ -196,8 +196,7 @@ ifeq ($(MFEM_USE_OPENMP),YES)
 endif
 
 # List of MFEM dependencies, that require the *_LIB variable to be non-empty
-MFEM_REQ_LIB_DEPS = METIS SIDRE LAPACK SUNDIALS MESQUITE SUITESPARSE SUPERLU\
- STRUMPACK GECKO GNUTLS NETCDF PETSC MPFR
+MFEM_REQ_LIB_DEPS = SUPERLU METIS SIDRE LAPACK SUNDIALS MESQUITE SUITESPARSE STRUMPACK GECKO GNUTLS NETCDF PETSC MPFR
```

Comment:
Why this change?

Reply:
SUPERLU uses METIS calls that are not used elsewhere. At least on LC, the link step requires that the dependencies be listed in order or else the extra functions are not included and the build fails.